restClient.http

```diff
@@ -1,7 +1,10 @@
 @baseUrl = http://localhost:3000
 
 ### GET podcast by term
-GET {{baseUrl}}/api/byTerm/space
+GET {{baseUrl}}/api/byTerm/bad crypto
```

Comment:
Not sue about the whitespace here `bad crypto`

Reply:
Oh thats just from my testing. It actually works. Vercel will automatically fill empty spaces with %20.